### PR TITLE
Issue #13345: Enable Local Final Variable Name Check Exampls Test

### PR DESCRIPTION
**Pull request by piyush-1234**
Issue: #13345

### CODE REVIEW

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/naming/localfinalvariablename/Example1.java [hunk 1]
```diff
@@ -6,5 +6,8 @@
 </module>
 */
 
+package com.puppycrawl.tools.checkstyle.checks.naming.localfinalvariablename;
+
 // xdoc section -- start
+class Example1{}
 // xdoc section -- end
```
Comment (romani) [inline src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/naming/localfinalvariablename/Example1.java:13]: Please do us a favor copy Example2 class content to Example1.
And add Example1 to template file.
Looks like we lost Example1 usage at all

**Review by romani — CHANGES_REQUESTED** [review]
Good, but one item to improve:

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/naming/localfinalvariablename/Example1.java (hunk 1 above)
Comment (piyush-1234) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/naming/localfinalvariablename/Example1.java:25]: done

**Review by romani — APPROVED** [review]
Ok to merge.

Thanks a lot for helping.

**Comment by piyush-1234** [comment]
> Ok to merge.
> 
> Thanks a lot for helping.

my pleasure.

**Pull request merged**